src/main/java/app/bpartners/geojobs/service/FeatureSurfaceService.java

```diff
@@ -0,0 +1,24 @@
+package app.bpartners.geojobs.service;
+
+import app.bpartners.geojobs.endpoint.rest.controller.mapper.FeatureMapper;
+import app.bpartners.geojobs.endpoint.rest.model.Feature;
+import app.bpartners.geojobs.model.parcelization.area.AreaComputer;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class FeatureSurfaceService {
+  private static final AreaComputer areaComputer = new AreaComputer();
```

Comment:
Constant variables must always be capital letter.

areaComputer > AREA_COMPUTER